src/components/data-table/_data-table.scss

```diff
@@ -148,6 +148,10 @@
     height: 12px;
     margin-left: rem(7px);
     margin-right: rem(2px);
+
+    @media all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm) {
```

Comment:
Mozilla specific hack, found here: http://browserhacks.com/. Checkboxes were misaligned in the data-table